indico/htdocs/js/indico/modules/registration/registration.js

```diff
@@ -108,6 +108,20 @@
         $('#select-none').on('click', function() {
             $('table.i-table input.select-row').prop('checked', false).trigger('change');
         });
+
+        $('.js-dialog-send-email').ajaxDialog({
+            onClose: function(data) {
+                if (data) {
+                    location.reload();
+                }
+            },
+            getExtraData: function() {
+                var identifiers = $('.registrations input:checkbox:checked').map(function() {
```

Comment:
in the agreement it contains identifiers, not ids (since unsent agreements don't have an ID since they are not in the DB)